lib/ecto/migrator.ex

```diff
@@ -756,4 +764,27 @@ defmodule Ecto.Migrator do
   defp after_action(_repo, :noop) do
     :noop
   end
+
+  use GenServer
+
+  def start_link(state) do
+    GenServer.start_link(__MODULE__, state, name: __MODULE__)
```

Comment:
```suggestion
  def start_link(opts) do
    GenServer.start_link(__MODULE__, opts, name: __MODULE__)
```

And please add some docs to this function documenting the existing options. (:migrator should be private for testing only).